README.MD

```diff
@@ -10,6 +10,7 @@ Fill in the following values:
 
 `CONTRACT_ADDRESS`: Your contract address, see details below
 `ACCOUNT_ADDRESS` : Your account address, see details below
+`USE_TESTRPC` : true if testRpc is used, otherwise false
```

Comment:
We don't need the USE_TESTRPC in the configuration, this is a native environment variable.

If you want to know if you're using testRpc, can use the utils.js file:
var utils = require('./utils');
utils.isTestRpc

I don't see that you use this anyway in the rest of the code, so this is probably leftovers from the previous PR.